### PR TITLE
CT-256 - renaming txbase64 to txhex in algo & trust custodial implementation

### PR DESCRIPTION
**Pull request by twtaylorbitgo**
We need to fixup Algo to be txHex instead of txBase64 (design decision) and then trust custodial implementation items

### CODE REVIEW

File: modules/core/src/v2/coins/algo.ts [hunk 1]
```diff
@@ -50,6 +45,8 @@ export interface TransactionPrebuild {
     genesisHash: string;
     note?: string;
   }
+  keys: string[],
```
[the remark this answers is not in the document]
Comment (twtaylorbitgo) [reply modules/core/src/v2/coins/algo.ts]: doneski

File: modules/core/src/v2/coins/algo.ts [hunk 2]
```diff
@@ -244,10 +239,22 @@ export class Algo extends BaseCoin {
...
   }
 
+  isStellarSeed(seed: string): boolean {
+    return stellar.StrKey.isValidEd25519SecretSeed(seed);
+  }
+
+  convertFromStellarSeed(seed: string): string {
+    // assume this is a trust custodial seed if its a valid ed25519 prv
+    if (!this.isStellarSeed(seed)) {
+      return null;
```
Comment (argjv) [inline modules/core/src/v2/coins/algo.ts]: Why not just return the same one you receive in the params? This way, this method will always return a valid seed, and there is no risk of hitting NPE

File: modules/core/src/v2/coins/algo.ts [hunk 3]
```diff
@@ -244,10 +239,22 @@ export class Algo extends BaseCoin {
...
-      memo
+      memo,
     };
   }
 
+  isStellarSeed(seed: string): boolean {
+    return stellar.StrKey.isValidEd25519SecretSeed(seed);
+  }
+
+  convertFromStellarSeed(seed: string): string {
```
Comment (argjv) [inline modules/core/src/v2/coins/algo.ts]: Docs pls

File: modules/core/src/v2/coins/algo.ts (hunk 2 above)
Comment (tylerlevine) [reply modules/core/src/v2/coins/algo.ts]: Good call, this should really throw an Error instead of returning null.

Comment (argjv) [reply modules/core/src/v2/coins/algo.ts]: https://bitgoinc.atlassian.net/browse/CT-630

**Pull request merged**